README.md

```diff
@@ -1,50 +1,50 @@
 # VKD3D-Proton
 
-VKD3D-Proton is a fork of VKD3D, which aims to implement the full Direct3D 12 API on top of Vulkan.
-The project serves as the development effort for Direct3D 12 support in [Proton](https://github.com/ValveSoftware/Proton).
+The VKD3D-Proton is a VKD3D fork designed to implement the entire Direct3D 12 API at the top of Vulcan.
+The project serves as a development effort to support Direct3D 12 in [Proton] (https://github.com/ValveSoftware/Proton).
 
 ## Upstream
 
-The original project is available at [WineHQ](https://source.winehq.org/git/vkd3d.git/).
+The original project is available at [WineHQ] (https://source.winehq.org/git/vkd3d.git/).
 
 ## Priorities
 
-Performance and compatibility are important targets, at the expense of compatibility with older drivers and systems.
-Modern Vulkan extensions and features are aggressively made use of to improve performance and compatibility.
-It is recommended to use the very latest drivers you can get your hands on for the best experience.
-Backwards compatibility with the vkd3d standalone API is not a goal of this project.
+Efficiency and compatibility are important goals at the expense of compatibility with older drivers and systems.
+Modern Vulcan extensions and features are being used aggressively to improve performance and compatibility.
+For the best experience, it is recommended to use the latest drivers that you get your hands on.
+Compatibility with the vkd3d standalone API is not the goal of this project.
 
 ------
 
 ## Cloning the repo
 
-To clone the repo you should run:
+To clone a turnip, run:
 ```
 git clone --recursive https://github.com/HansKristian-Work/vkd3d-proton
 ```
-in order to pull in all the submodules which are needed for building.
+to pull all the submodules needed for construction.
 
 ## Building VKD3D
 
 ### Requirements:
 - [wine](https://www.winehq.org/) (for `widl`) [for native builds]
-  - On Windows this may be substituted for [Strawberry Perl](http://strawberryperl.com/) as it ships `widl` and is easy to find and install -- although this dependency may be eliminated in the future.
+  - On Windows this may be substituted for [Strawberry Perl](http://strawberryperl.com/) as it sends a `widl` and is easy to find and install - although this dependency can be fixed in the future.
```

Comment:
No, it *includes* -wildl-.
Removal of a dependency ('may be eliminated') is *very* different from it being 'fixed'.